test/foundry/CollateralAgreement.t.sol

```diff
@@ -266,28 +266,40 @@ contract CollateralAgreementTest is AgreementFrameworkTestBase {
         settlement[0] = PositionParams(bob, 3 * 1e18);
 
         hevm.prank(arbitrator);
-        hevm.expectRevert(abi.encodeWithSignature("MissingPositions()"));
+        hevm.expectRevert(abi.encodeWithSignature("PositionsMustMatch()"));
         framework.settleDispute(disputedId, settlement);
     }
 
-    function testSettlementMustMatchBalance() public {
+    function testSettlementCantAddNewPositions() public {
         bytes32 disputedId = _setupDispute();
 
         PositionParams[] memory settlement = new PositionParams[](3);
-        // Settlement balance = 4 > 3 = agreement balance
         settlement[0] = PositionParams(bob, 1.5 * 1e18);
-        settlement[1] = PositionParams(alice, 1 * 1e18);
+        settlement[1] = PositionParams(alice, 0 * 1e18);
         settlement[2] = PositionParams(address(0xD011), 1.5 * 1e18);
 
         hevm.prank(arbitrator);
-        hevm.expectRevert(abi.encodeWithSignature("SettlementBalanceMustMatch()"));
+        hevm.expectRevert(abi.encodeWithSignature("PositionsMustMatch()"));
+        framework.settleDispute(disputedId, settlement);
+    }
+
+    function testSettlementMustMatchBalance() public {
+        bytes32 disputedId = _setupDispute();
+
+        PositionParams[] memory settlement = new PositionParams[](2);
+        // Settlement balance = 4 > 3 = agreement balance
```

Comment:
I didn't understand "Settlement balance = 4 > 3 = agreement balance."  What does "4 > 3" mean?

Reply:
Yep, I'm going to make it clear in the comment